### DISCUSSION TITLE
[feature] Keyboard shortcuts

### DISCUSSION THREAD

**Discussion by E-Bismuth** [OpenHantek/OpenHantek6022#287] [Ideas]
Hi, it will be great if the software could accept keyboard shortcuts, like that we could plug a macro keyboard so we could use it like a "regular" oscilloscope with buttons and potentiometers

**Comment by BalintPogatsa** [comment]
This Arcade USB encoder was recommended in a youtube video about creating your own button box.
https://www.amazon.com/gp/product/B07GJK2G1F/ref=ppx_yo_dt_b_asin_title_o03_s00?ie=UTF8&th=1

https://www.youtube.com/watch?v=Kx3-JhT4X_M&ab_channel=OverKillSimulations

Maybe one could take the front panel of an old oscilloscope and convert it to an OpenHantek controller...
